src/args.rs

```diff
@@ -99,6 +99,16 @@ pub struct Args {
     )]
     pub exclude: Regex,
 
+    /// HTTP basic authentication credentials
+    #[structopt(
+        short,
+        long,
+        use_delimiter = true,
+        value_delimiter = " ",
+        help = "HTTP basic authentication credentials space-separated as \"username password host\". Can be repeated for multiple credentials"
```

Comment:
The help is a little unclear on how to use the option. When reading it, I thought that you had to give `--auth UW PW H` multiple times. Would it be possible to improve the wording a little so that the user understands you have to use `--auth UW1 PW1 H1 uw2 pw2 h2`?